Fabric.Authorization.Persistence.CouchDb/Services/CouchDbAccessService.cs

```diff
@@ -62,9 +61,9 @@ public CouchDbAccessService(ICouchDbSettings config, ILogger logger)
             {
                 if (!(await client.Database.GetAsync()).IsSuccess)
                 {
-                    _logger.Information($"could not retrieve database information. attempting to create. Current Thread Id: {Thread.CurrentThread.ManagedThreadId}");
+                    //_logger.Information($"could not retrieve database information. attempting to create. Current Thread Id: {Thread.CurrentThread.ManagedThreadId}");
                     var creation = await client.Database.PutAsync();
-                    _logger.Information($"database created if it did not exist. Current Thread Id: {Thread.CurrentThread.ManagedThreadId}");
+                    //_logger.Information($"database created if it did not exist. Current Thread Id: {Thread.CurrentThread.ManagedThreadId}");
                     if (!creation.IsSuccess)
```

Comment:
did you intend to leave those two statements commented out?

Reply:
The .NET Standard Library does not include System.Threading.Thread namespace / DLL by default. I commented out to get it to build, but I will see if there is a way around this.

Reply:
ahh ok. nbd just curious